package.json

```diff
@@ -17,6 +17,7 @@
     "@heroicons/react": "^2.1.5",
     "@react-google-maps/api": "^2.19.3",
     "axios": "^1.7.5",
+    "bcryptjs": "^2.4.3",
```

Comment:
Why was this dependency added? I don't see this used anywhere

Reply:
Bumping this - do we need this dependency for anything?

Reply:
It's for back so I think I don't need it in my commit

Reply:
And I don't see it now in package.json